pytorch_to_returnn/torch/_C.py

```diff
@@ -122,17 +123,48 @@ class Size(tuple):  # type: Tuple[SizeValue, ...]
 
 class SizeValue(int):
   """
-  We extend this, to store extra information, e.g. such that this reflects the batch-dim.
+  We extend this, to store extra information, e.g. corresponding RETURNN dim tags.
   """
-  is_batch_dim: bool = False
-  merged_dims: list[SizeValue] = []
+  def __new__(cls, x: int, dim_tag: Optional[Dim] = None, merged_dims: Optional[List[SizeValue]] = None, tensor=None, axis=None):
```

Comment:
`tensor` should be typed.
`axis` should not be needed.

Reply:
I think we also should rename `tensor` to `originating_tensor` or so.

Reply:
For typing of `tensor` we would need to import `Tensor` which leads to a circular import, right?

Reply:
No, it doesn't need to be imported. You should never import sth just for typing, or maybe only conditionally with `if typing.TYPE_CHECKING:`. You probably should have `from __future__ import annotations` if Python/PyCharm complains.